Source/WebUI/Static/js/lang.js

```diff
@@ -0,0 +1,81 @@
+// lang.js
+// By GoBlock2021
+
+
+// 定义默认语言
```

Comment:
Can we have the commentary here in english please? It makes maintaining things easier in the long run.

Reply:
Okay, I've translated them into English